src/web/components/ClickToView.tsx

```diff
@@ -12,8 +12,11 @@ import { Lines } from '@guardian/src-ed-lines';
 
 type Props = {
 	children: React.ReactNode;
-	width: number;
-	height: number;
+	width?: number;
+	height?: number;
```

Comment:
Given that we dont think we can easily match the size of the overlay to the size of the embed it 'covers', I am now using width: 100% and not setting a height.

You had some concern that not setting the height would cause a double reflow ie when the over lay was removed and when the embed is inserted.

This doesnt seem to be the case, see:

https://5dfcbf3012392c0020e7140b-ymnejlgtya.chromatic.com/?path=/story/components-clicktoview--small-story

Not having a height definitely simplifies things, what do you think?